net-misc/r8152/metadata.xml

```diff
@@ -9,9 +9,15 @@
 		<email>proxy-maint@gentoo.org</email>
 		<name>Proxy Maintainers</name>
 	</maintainer>
-	<longdescription>Official Realtek r8152 linux driver. The following cards are currently supported:
-RTL8156 RTL8156B(S)(G) RTL8153 RTL8153B RTL8154 RTL8154B RTL8152B</longdescription>
+	<longdescription>
+		Official Realtek r8152 linux driver. Supports most realtek
+		USB network cards, including: RTL8157, RTL8156(,B), RTL8153(,B,C,D,E),
+		RTL8154(,B), RTL8152B.
+	</longdescription>
 	<use>
 		<flag name="center-tap-short">Enable support for center tap short</flag>
 	</use>
+	<upstream>
+		<remote-id type="github">jayofdoom/realtek-r8152-unchanged</remote-id>
```

Comment:
if desired, happy to give any gentoo developers I know maintainer access on this to reduce bus factor

Reply:
I'm not super worried one way or another, though you're welcome to give me access. (I don't use this module but being a backup point of contact is basically easy.)